src/constants/localizacion.js

```diff
@@ -0,0 +1,7 @@
+const localizacionAlbarracin = {
+    latitud : 40.36,
+    longitud : -1.54,
+  };
+
```

Comment:
Creo que lo mejor es quitar estos comentarios del código y los podemos poner en la descripción de la pullRequest. También habría que cambiar el nombre de la variable y del archivo por AlbarracinCoordinates y de los atributos por longitude y latitude

Reply:
Mantener la consistencia tiene bastante importancia, pero en ese caso tendríais que cambiar también el nombre del fichero.
Por otro lado, insisto, ¿el problema es el nombre de las claves o el hecho de que se haya definido una constante con una ciudad específica? ¿El modelo del problema es que cada ciudad de la que vayas a preguntar necesita que se defina un código? ¿Si un cliente pide añadir otra localización, tenéis que cambiar todo el código?

Reply:
Es una variable de ejemplo. Sólo necesaria para saber que necesitaremos recibir unas coordenadas gps de la zona de escalada

Reply:
Si eso es lo que necesitáis, haced eso. Recordad el lenguaje ubicuo. Tendrá que haber una clase "ZonaDeEscalada" con las propiedades que suela tener (supongo que coordenadas, quizás alguna más, ¿no? Siempre que lo requiera la HU, igual es sólo la coordenada y el nombre o algo)